### PR TITLE
fix: incorrect height for quickpanel's subplugin

### PR DESCRIPTION
**Pull request by 18202781743**
Item's height is set by StackView when pushed a Component, so we add a contentHeight as Item's implicitHeight, and we rebind StackView's height when StackView.activating.

### CODE REVIEW

**Comment by hudeng-go** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-shell/pull/465#" title="Author self-approved">18202781743</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-shell).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-shell/blob/develop/tray/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by hudeng-go** [comment]
### deepin pr auto review
关键摘要:
- 在`QuickPanelPage.qml`中，`StackView`的高度属性被设置为`panelView.contentHeight`，这可能会导致布局问题，因为`StackView`的高度应该是固定的，而不是依赖于其子项的高度。
- 在`SubPluginPage.qml`中，`implicitHeight`属性被设置为`contentHeight`，这可能会导致布局问题，因为`Item`的`height`应该是固定的，而不是依赖于其子项的高度。
- 在`SubPluginPage.qml`中，`ShellSurfaceItem`被替换为`ShellSurfaceItemProxy`，这可能会影响与ShellSurface的交互。需要确认这是否是预期的更改。

是否建议立即修改:
是

- 应该立即修复`QuickPanelPage.qml`中的布局问题，以确保`StackView`的高度是固定的。
- 应该立即修复`SubPluginPage.qml`中的布局问题，以确保`Item`的高度是固定的。
- 如果`ShellSurfaceItemProxy`的替换不是预期的更改，应该立即恢复`ShellSurfaceItem`。

**Review by yixinshark — APPROVED** [review]

**Comment by hudeng-go** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-shell/pull/465#" title="Author self-approved">18202781743</a>*, *<a href="https://github.com/linuxdeepin/dde-shell/pull/465#pullrequestreview-2132214962" title="Approved">yixinshark</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-shell).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-shell/blob/develop/tray/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
